#request
float64[] CPCmd
int8 cpMode
---
#response
bool success
